src/ViaPhone.php

```diff
@@ -251,48 +153,29 @@ public function getDevice($phoneNumber)
 		return null;
 	}
 
-	/**
-	 * @param array $params
-	 * @return array
-	 * @throws \Exception
-	 */
+
 	public function getDevices(array $params = [])
```

Comment:
mozna criterias? u getRecords to pojmenovavame jako criterias ne? je to to stejne?

Reply:
done